PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-azure-sdk/src/main/java/com/microsoft/azure/toolkit/intellij/azure/sdk/buildtool/RuleConfig.java

```diff
@@ -84,10 +88,22 @@ String getAntiPatternMessage() {
 
     /**
      * This method returns the map of discouraged identifiers
+     * This is used for base classes that have a set of discouraged identifiers
+     * and a corresponding set of antipattern messages.
      *
      * @return Map of discouraged identifiers
      */
```

Comment:
What is the key/value pair for this mapping?

Reply:
There's 2 existing uses of it:
Client Name: Anti-pattern
API Name: Anti-pattern